transforms/code/license_copyright_removal/ray/README.md

```diff
@@ -0,0 +1,46 @@
+# License copyright removal 
+
+Please see the set of
+[transform project conventions](../../../README.md)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary
+This module is designed to detect and remove license and copyright information from code files. It leverages the [ScanCode Toolkit](https://pypi.org/project/scancode-toolkit/) to accurately identify and process licenses and copyrights in various programming languages.
+
+After detecting license and copyright position code has been stored at same column. Now lines which doesn't contain license or copyright copied to same position.
+
+## Configuration and command line Options
+
+The set of dictionary keys holding configuration for values are as follows:
+
+* --license_copyright_removal_contents_column_name - specifies the column name which holds code content. By default the value is 'contents'.
```

Comment:
These are the CLI parameters, not the configuration keys provided to the transform's init().  Both should be discussed.

Reply:
You have correctly (IMHO) defined the configure keys to not include the cli prefix (i.e. contents_column_name vs license_copyright_contents_column_name).